src/routes/backpack/logmanager.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Backpack\LogManager Routes
+|--------------------------------------------------------------------------
+|
+| This file is where you may define or change all of the routes that are
```

Comment:
Technically we're not really changing anything here - it's just defining everything.